examples/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM jupyter/minimal-notebook
+
+ARG PYTHON_VERSION=3.9
+ARG SCALA_VERSION=2.12
+ARG DELTA_LAKE_VERSION=1.1.0
+ARG SPARK_VERSION=3.2.1
+ARG DELTA_PACKAGE=io.delta:delta-core_${SCALA_VERSION}:${DELTA_LAKE_VERSION}
+
+USER root
+
+RUN apt-get update && \
+    apt-get -y install openjdk-8-jdk r-base \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+USER $NB_UID
+
+ENV JUPYTER_TOKEN="docker"
+ENV JUPYTER_ENABLE_LAB="yes"
+
+RUN pip install delta-spark==${DELTA_LAKE_VERSION}
+
+RUN pip install jupytext
+
+COPY --chown=$NB_UID:$NB_UID python python/
```

Comment:
are you sure that these python examples work through jupyter?

Reply:
Can you update DELTA_LAKE_VERSION to 2.1.x? Thanks!